### ISSUE TITLE
Version 2.0.10 problem "github.com/savsgio/gotils"

### ISSUE THREAD

**Issue by angelogeminiani** [gofiber/websocket#66]
Compiling I have an error depending on fasthttp/websocket 1.4.3 because it uses github.com/savsgio/gotils that give me this error:
Build constraints exclude all the Go files in '/Users/angelogeminiani/go/pkg/mod/github.com/savsgio/gotils@v0.0.0-20210907153846-c06938798b52'

**Comment by ReneWerner87** [comment]
have tested it in a test application
-> get no errors


can you share your go.mod file and the go.sum file and if necessary the go mod command you use

**Comment by ReneWerner87** [comment]
with go get i currently have this problem too, but i can't find the cause, maybe we have to create a bug in the fasthttp/websocket repository

**Comment by angelogeminiani** [comment]
have same problem even creating a new project.
Just run:
```
go get -u github.com/gofiber/fiber/v2
go get -u github.com/gofiber/websocket/v2
```

Problem is using go get -u

If I try only `go get github.com/gofiber/websocket/v2` it works.

**Comment by ReneWerner87** [comment]
![image](https://user-images.githubusercontent.com/7063188/132854427-0dba4287-a769-4f3b-97d9-e30809da9aba.png)

![image](https://user-images.githubusercontent.com/7063188/132854473-4b5ccdfd-b666-4076-ae85-82f21defcd6f.png)
can do nothing about it

this dependencie 
![image](https://user-images.githubusercontent.com/7063188/132854598-da1468c2-754a-4510-9993-0e3744e54547.png)

gotils is part of 
fasthttp/websocket
![image](https://user-images.githubusercontent.com/7063188/132854702-bced9f8e-2d2c-412a-b6ab-1500861b8459.png)

**Comment by G2G2G2G** [comment]
Got this running `go get -u`  (I run that and then `go mod tidy` for updates) 
`github.com/savsgio/gotils: cannot find module providing package github.com/savsgio/gotils`

but it doesn't seem to do anything, my programs work fine still and compile...

**Comment by efectn** [comment]
fixed with v2.0.15.
